config/fulcio-config.yaml

```diff
@@ -12,25 +12,29 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: fulcio-config
-  namespace: fulcio-dev
 data:
-  config.json: |
-    {
-      "OIDCIssuers": {
-        "https://accounts.google.com": {
-          "IssuerURL": "https://accounts.google.com",
-          "ClientID": "sigstore",
-          "Type": "email"
-        },
-        "https://oauth2.sigstore.dev/auth": {
-          "IssuerURL": "https://oauth2.sigstore.dev/auth",
-          "ClientID": "sigstore",
-          "Type": "email"
+    config.json: |-
+        {
+          "OIDCIssuers": {
+            "https://accounts.google.com": {
+              "IssuerURL": "https://accounts.google.com",
+              "ClientID": "sigstore",
+              "Type": "email"
+            },
+            "https://oauth2.sigstore.dev/auth": {
+              "IssuerURL": "https://oauth2.sigstore.dev/auth",
+              "ClientID": "sigstore",
+              "Type": "email"
+            },
+            "https://oidc.dlorenc.dev": {
+              "IssuerURL": "https://oidc.dlorenc.dev",
+              "ClientID": "sigstore",
+              "Type": "spiffe"
+            }
+          }
         }
```

Comment:
As this is the main default server config, could there be a fulcio-federated-config.yaml that is used for other specific providers? It could under federated/config maybe?

Reply:
Moved it down a directory! How's this?

Reply:
I can't see that, it still seems to be in the main config.json

Reply:
So this file is actually generated now (and there's a test to make sure its up to date). The full merged list needs to be in one place. The user-edited ones are now in federation/ and federation/external.